tests/components/dataset_activity/test_views.py

```diff
@@ -138,11 +254,11 @@ async def test_list_dataset_activities_returns_dataset_activities_filtered_by_ac
         received_ccs = body('.result[].container_code').all()
         received_total = body('.total').first()
 
-        assert received_ccs == [str(dataset_activity.container_code)]
-        assert received_total == 1
+        assert set(received_ccs) == {str(dataset_activity.container_code), str(item_activity.container_code)}
```

Comment:
Please remove `str` since `container_code` is a string field.
```suggestion
        assert set(received_ccs) == {dataset_activity.container_code, item_activity.container_code}
```